source/docs/prismic/routable-types.md

```diff
@@ -179,51 +177,48 @@ Album: {
     const baseUrl = (await loaders.Shop.getCurrentShop()).baseUrl,
     return `{baseUrl}/${content.url}`;
   },
-+    priority: () => 1,
-+    seoImages: (content) => [],
-+    lastmod: () => "2020-01-01T12:00:00.000Z",
-+    changefreq: () => "daily",
++ priority: () => 1,
++ seoImages: (content) => [],
++ lastmod: () => "2020-01-01T12:00:00.000Z",
++ changefreq: () => "daily",
 },
 };
-```
-
-2. Enable sitemap in the module definition file:
-
-```diff
+  ```
+1. Enable sitemap in the module definition file:
```

Comment:
Why did you change for several "first point" in a list of steps ?

Reply:
so that the markdown parser automatically adds the right number and the resulting HTML markup is a proper numbered list, see https://deploy-preview-301--heuristic-almeida-1a1f35.netlify.app/docs/prismic/routable-types.html#Add-a-routable-custom-type-to-the-sitemap for this example

Also, if we have to reorder/insert/remove a list item, you don't have to worry about the number.

Reply:
Ok, I didn't know that it worked that way